README.md

```diff
@@ -498,6 +498,80 @@ Flag | Notes
 `--https_proxy`      | Needed for package manager or Ubuntu guest and for from github downloaded packages
 `--ftp_proxy`       | Needed for some Perfkit packages
 
+CONFIGURATIONS AND CONFIGURATION OVERRIDES
+==================
+Each benchmark now has an independent configuration which is written in YAML.
+A user config is a way to override this default configuration on a per benchmark
+basis. This allows for much more complex setups than previously possible,
+including running benchmarks across clouds.
+
+A benchmark configuration has a somewhat simple structure. It is essentially
+just a series of nested dictionaries. At the top level, it contains VM groups.
+VM groups are logical groups of homogenous machines. The VM groups hold both a
+"vm_spec" and a "disk_spec" which contain the parameters needed to
+create members of that group. Here is an example of an expanded configuration:
+```
+iperf:
+  vm_groups:
+    iperf_vm_1:
+      vm_count: 1
+      vm_spec:
+        GCP:
+          machine_type: n1-standard-1
+          image: ubuntu-14-04
+          zone: us-central1-c
+        AWS:
+          machine_type: m3.medium
+          image: ami-######
+          zone: us-east-1a
+        # Other clouds here...
+    iperf_vm_2:
+      vm_count: 1
+      # This specifies the cloud to use for the group. This allows for
+      benchmark configurations that span clouds.
+      cloud: AWS
+      vm_spec:
+        GCP:
+          machine_type: n1-standard-1
+          image: ubuntu-14-04
+          zone: us-central1-c
+        # Other clouds here...
+      disk_count: 0
```

Comment:
Yes - iperf doesn't need the disk_spec, but I wanted to illustrate the use of as many keys as possible.